Dockerfile

```diff
@@ -1,21 +1,25 @@
 # Use an ARG to select which build target to compile and use
 ARG TARGET_BUILD=standard
-ARG BINARY_NAME=kube-network-policies-${TARGET_BUILD}
 
 FROM --platform=$BUILDPLATFORM golang:1.24 AS builder
 WORKDIR /src
+
+# Get target architecture for cross-compilation
+ARG TARGETOS
+ARG TARGETARCH
+ARG TARGET_BUILD
+
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
 
-# Build the specific binary based on the build argument
-ARG TARGET_BUILD
-RUN make build-${TARGET_BUILD}
```

Comment:
I do not want to drift, this makes the makefile and the dockerfile to diverge

Reply:
Trying again to have them aligned

Reply:
it was cross compiling before for arm64 so I do not think the problem should be here

Reply:
aligned the makefile

i will try to see again if the changes are needed, or if it was my mistake